setup.py

```diff
@@ -174,10 +174,14 @@ def vitables_data_files():
 # main stuff #
 # ========== #
 
+execfile('./liam2/version.py')
+# now we have a `__version__` variable
+
+
 setup(
     name="liam2",
     # cx_freeze wants only ints and dots (full version number)
-    version=int_version('0.11.0'),
+    version=int_version(__version__),

```

Comment:
`__version__` probably needs to pass through long_release_name (from make_release). hmmm, now that I think of it, it would be better to instead change the comment that I just commented on and use "short" (from make_release) at https://github.com/liam2/liam2/blob/master/liam2/main.py#L345